cmd/agent/api/agent/agent.go

```diff
@@ -367,6 +370,23 @@ func getTaggerList(w http.ResponseWriter, r *http.Request) {
 	w.Write(jsonTags)
 }
 
+type tagRequest struct {
+	Entity string
+}
+
+func getTags(w http.ResponseWriter, r *http.Request) {
+	// tagger should be init
+	request := tagRequest{}
+	err := json.NewDecoder(r.Body).Decode(&request)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+	tags, _ := tagger.Tag(request.Entity, collectors.HighCardinality)
```

Comment:
Shouldn't we make cardinality a request parameter? I believe not all agents may need high cardinality for all entities?

Reply:
I forgot cleaning this up - we should only have the hostname endpoint in this PR. Thanks for spotting this.